### DISCUSSION TITLE
NGINX反向代理后无法找到文件

### DISCUSSION THREAD

**Discussion by TerryWang6** [requarks/wiki#5680] [Error / Bug Report]
例如我域代理地址：a.com/helpDoc/
项目实际访问地址：a.com
只到a.com寻找项目文件运行。
项目中怎么在端口号后面添加一个目录呀，找了一天也没找到。

**Comment by cdn0x12** [comment]
Don't understand what you mean.
If you are trying to access your wiki via a custom port, then try `http(s)://a.com:<port>/helpDoc/`
If you're running your wiki successfully but unable to access `a.com` via your nginx reverse-proxy, please post your nginx configuration for further inspection.
If you can access `a.com` but can't access `a.com/helpDoc/`, please check if there are other nginx rules that conflict with your url or if the `helpDoc` is the correct url.
If you are attempting to reverse-proxy the whole wiki instance behind `a.com/helpDoc/`: that's not possible at the moment, try hosting it in `helpdoc.a.com` instead.
